README.md

```diff
@@ -0,0 +1,81 @@
+# Clearwater FV Test
+
+This repository contains "Functional Verification" tests for Clearwater. These
+are tests that do not fit well into per-project unit tests. This might be
+because they use multiple components or external services / databases (for
+example).
+
+## Dependencies
+
+The FV tests depend on a number of tools and libraries.  Some of these are
+included as git submodules, but the rest must be installed separately.
+
+On Ubuntu 12.04,
+
+1.  update the package list
+
+        sudo apt-get update
+
+2.  install the required packages
+
+        sudo apt-get install git cmake make gcc g++ bison flex libsctp-dev libgnutls-dev libgcrypt-dev libidn11-dev libtool autoconf libboost-dev libboost-test-dev automake libssl-dev
```

Comment:
Good point! I will spin up a new VM and check that this list of packages is right.